pypeit/core/coadd.py

```diff
@@ -2082,24 +2086,25 @@ def combspec(waves, fluxes, ivars, masks, sn_smooth_npix,
         percentile fraction cut used for selecting minimum SNR cut for robust_median_ratio
     maxiter_scale: int, optional, default=5
         Maximum number of iterations performed for rescaling spectra.
+    wave_grid_input (`numpy.ndarray`_):
```

Comment:
I think this is the wrong doc style which will break on readthedocs

Reply:
I need help here.